### PR TITLE
SV-467 Assessor outer API to return all standards

### PR DESCRIPTION
**Pull request by DevenKShah**
Original PR #299 was already approved and merged into SV-372. However this was required to be separated as it is not going out in the same release as SV-372. 

More work was done which is going out with SV-372, hence had to rebase the branch on top of StandardOptions release branch and reopen the PR. 

I have excluded postman changes as they become too difficult to merge. I will update them in a newer PR once this PR is merged into a release branch.

### CODE REVIEW

**Comment by DevenKShah** [comment]
This branch is based off StandardOptions, to keep up with latest changes, however it is not going out in the release so cannot merge this yet. Once StandardOptions is merged in to master, this will have to rebase off master.

File: src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs
Comment (CodeWolf86) [inline src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs:15]: Do these need updating based on the updates by @ben1stone  ?

File: src/SFA.DAS.Assessors.Api/Models/GetStandardDetailsResponse.cs
Comment (CodeWolf86) [inline src/SFA.DAS.Assessors.Api/Models/GetStandardDetailsResponse.cs:48]: Do we need to update these fields due to the changes to add "Proposed" ?

File: src/SFA.DAS.Assessors/Application/Queries/GetStandardDetails/GetStandardDetailsQueryHandler.cs
Comment (CodeWolf86) [inline src/SFA.DAS.Assessors/Application/Queries/GetStandardDetails/GetStandardDetailsQueryHandler.cs:25]: Given the changes to the Courses Api, Can we consolidate the Query for the get by ID request into one handler now?

File: src/SFA.DAS.Assessors/InnerApi/Responses/StandardDetailResponse.cs
Comment (CodeWolf86) [inline src/SFA.DAS.Assessors/InnerApi/Responses/StandardDetailResponse.cs:48]: Do we need to update these fields due to the changes to add "Proposed" ?

File: src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetStandardDetailsByStandardUIdRequest.cs
Comment (CodeWolf86) [inline src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetStandardDetailsByStandardUIdRequest.cs:5]: Can we merge this with the requests that operate by LarsCode?

File: src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs
Comment (CodeWolf86) [inline src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs:62]: Bad Request or Internal Server Error?

**Review by CodeWolf86 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs
Comment (DevenKShah) [reply src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs:15]: yes... will have to wait till that is merged... I am too far from those changes :-(

File: src/SFA.DAS.Assessors.Api/Models/GetStandardDetailsResponse.cs
Comment (DevenKShah) [reply src/SFA.DAS.Assessors.Api/Models/GetStandardDetailsResponse.cs:48]: yes... will have to wait till that is merged... I am too far from those changes :-(

File: src/SFA.DAS.Assessors/Application/Queries/GetStandardDetails/GetStandardDetailsQueryHandler.cs
Comment (DevenKShah) [reply src/SFA.DAS.Assessors/Application/Queries/GetStandardDetails/GetStandardDetailsQueryHandler.cs:25]: I started doing this, but then I stopped coz I thought your story will hit this before me.

File: src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs
Comment (DevenKShah) [reply src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs:62]: ISE for sure... but has to be done at a later stage to avoid conflicts.

File: src/SFA.DAS.Assessors/InnerApi/Responses/StandardDetailResponse.cs
Comment (DevenKShah) [reply src/SFA.DAS.Assessors/InnerApi/Responses/StandardDetailResponse.cs:48]: yes... will have to wait till that is merged... I am too far from those changes :-(

File: src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetStandardDetailsByStandardUIdRequest.cs
Comment (DevenKShah) [reply src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetStandardDetailsByStandardUIdRequest.cs:5]: you do it 😉

File: src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs
Comment (CodeWolf86) [reply src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs:15]: Yeah that's fair, it's hard to visualise

**Review by CodeWolf86 — APPROVED** [review]

**Pull request merged**
